src/nccl_ofi_rdma.c

```diff
@@ -4967,6 +5359,171 @@ static inline int fini_bounce_buffers(nccl_net_ofi_rdma_ep_t *ep)
 	return ret;
 }
 
+static int get_mr_key(nccl_net_ofi_device_t *base_dev, void* mhandle,
```

Comment:
Why are we not checking against `FI_KEY_NOTAVAIL` and returning an error?

Reply:
Ahh. The man page I used was too old and did not list `FI_KEY_NOTAVAIL`. I'll check error!

Reply:
Applied.